evadb/server/command_handler.py

```diff
@@ -42,9 +43,16 @@ def execute_query(
     plan_generator = kwargs.pop("plan_generator", PlanGenerator(evadb))
     with query_compile_time:
         stmt = Parser().parse(query)[0]
-        StatementBinder(StatementBinderContext(evadb.catalog)).bind(stmt)
-        l_plan = StatementToPlanConverter().visit(stmt)
-        p_plan = plan_generator.build(l_plan)
+
+        # For certain statements, we plan to omit binder and optimizer to keep the code
+        # clean. So, we handle such cases here and pass the statement directly to the
+        # executor.
+        if stmt not in SKIP_BINDER_AND_OPTIMIZER_STATEMENTS:
```

Comment:
@xzdandy @jiashenC Added a skipping behavior in the system to avoid having to write boilerplate code for binder, optimizer.